spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraSQLContext.scala

```diff
@@ -37,6 +41,152 @@ class CassandraSQLContext(sc: SparkContext) extends SQLContext(sc) {
   @transient
   val conf = sc.getConf
 
+  @transient
+  private val clusterReadConf: Map[String, SparkConf] = Map()
```

Comment:
Shorter version would be:
```scala
private val clusterReadConf = Map[String, SparkConf]()
```
one other nit - in Scala we usually add `mutable.` prefix for mutable collections.